fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomes.java

```diff
@@ -107,4 +107,30 @@ public static void addBiomeVariant(Biome replaced, Biome variant, double chance,
 	public static void setRiverBiome(Biome parent, Biome river) {
 		InternalBiomeData.setOverworldRiverBiome(parent, river);
 	}
+
+	/**
+	 * Adds the biome as a large edge biome of the parent biome, with the specified weight.
```

Comment:
"Large edge" is IMO a poor term, these are primarily local transition biomes vs the other kind (shore biomes) - probably an issue for yarn though. They are larger than short biomes due to extra zooming, but that is IMO not their main distinct feature.

Reply:
So should I rename this to "transition biomes"